ButterSample/ButterSample/SampleEndpoints/BungheadsApplyRequest.swift

```diff
@@ -5,13 +5,13 @@
 import Foundation
 import Butter
 
-struct SampleRequestBodyB: Encodable {
+struct JasonDeruloBunghead: Encodable {
 	let first = "jason"
 	let last = "derulo"
 	let occupation = "unknown"
 }
 
-struct SampleEndpointB: Endpoint {
+struct BungheadsApplyRequest: Endpoint {
```

Comment:
for the sake of picking nits

if you have an `Endpoint` protocol, why not keep the `...Endpoint` suffix? imo `Endpoint` is more descriptive of the purpose of the protocol (and the conforming objects) than `...Request`, since you don't actually get the URLRequest from the `...Request` objects.

Reply:
makes sense 👍